### PR TITLE
Changed the improper css import and 'ember-moment' dependency

### PR DESCRIPTION
**Pull request by SamvelRaja**
Improper import available in the index.js 

screen shot 2015-09-17 at 5 51 53 pm

Here it is,

screen shot 2015-09-17 at 5 53 02 pm

### CODE REVIEW

**Comment by SamvelRaja** [comment]
@josemarluedke I couldn't find a reference for the use of 'ember-moment' here, but as it is breaking the addon. I am removing it from the package.
**Before:**
screen shot 2015-09-17 at 5 57 20 pm

**After**
screen shot 2015-09-17 at 6 13 16 pm

If you have anyother solution apart from removing it.

**Comment by SamvelRaja** [comment]
Feel free to reject this PR if not needed.

**Comment by SamvelRaja** [comment]
We are having one more dependency which will be added by ember-moment in its blue print.

[https://github.com/stefanpenner/ember-moment/blob/master/blueprints/ember-moment/index.js#L11](https://github.com/stefanpenner/ember-moment/blob/master/blueprints/ember-moment/index.js#L11)

File: index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
     this._super.included(app);
     this.app.import(app.bowerDirectory + '/moment/moment.js');
     this.app.import(app.bowerDirectory + '/bootstrap-daterangepicker/daterangepicker.js');
-    this.app.import(app.bowerDirectory + '/bootstrap-daterangepicker/daterangepicker.css');
+    this.app.import(app.bowerDirectory + '/bootstrap-daterangepicker/daterangepicker-bs3.css');
+    this.app.import(app.bowerDirectory + '/bootstrap-daterangepicker/daterangepicker-bs2.css');
```
Comment (josemarluedke) [inline index.js]: I think you should add only one of them, maybe just keep the bs3 option?

File: bower.json [hunk 2]
```diff
@@ -12,6 +12,8 @@
     "jquery": "^1.11.1",
     "loader.js": "ember-cli/loader.js#3.2.0",
     "qunit": "~1.17.1",
-    "bootstrap-daterangepicker": "~1.3.20"
+    "bootstrap-daterangepicker": "~1.3.20",
+    "moment": ">= 2.8.0",
+    "moment-timezone": ">= 0.1.0"
```
Comment (josemarluedke) [inline bower.json]: Is the timezone required?

Comment (SamvelRaja) [inline bower.json]: yes, @josemarluedke it has been added by the `ember-moment` lets not break the inner dependencies.

File: index.js (hunk 1 above)
Comment (SamvelRaja) [inline index.js]: :+1:

**Comment by josemarluedke** [comment]
Thank you @SamvelRaja!

**Pull request merged**